cf-apps/go/main.go

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2019 The Homeport Team
```

Comment:
Like discussed, move this file into a new directory in the root of the project called `assets`.